src/config/error.rs

```diff
@@ -21,7 +21,7 @@ impl TomlHelper for toml::Value {
             None => return,
         };
         for (key, _) in table.iter() {
-            if options.iter().find(|&entry| entry == key).is_none() {
+            if options.iter().any(|&entry| entry == key) {
```

Comment:
Aren't you missing a `!` here?

```suggestion
            if !options.iter().any(|&entry| entry == key) {
```

But slices also have a contains method:
```suggestion
            if !options.contains(&key.as_str()) {
````

Reply:
Ah right... I probably rushed the clippy pass and it didn't show up when I regen `gstreamer-rs`.